### PR TITLE
Use sass for styles

### PR DESCRIPTION
**Pull request by CYBAI**
Scss files under `styles` folder are compiled from `public/less`.

### CODE REVIEW

File: src-react/containers/path.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux'
 
+import "../styles/styles.scss";
```
Comment (AllenFang) [inline src-react/containers/path.js]: it's wired... why eslint allow double quotes

Comment (CYBAI) [reply src-react/containers/path.js]: Oh... I didn't notice that.
Let me fix it.

BTW, seems that the `react-app` eslint rules allow both `no-semi` and `semi`.
(Check line 2)

Comment (CYBAI) [reply src-react/containers/path.js]: Updated for single quotes; I also add the `webpack` alias for styles.

Besides, let's fix the eslint issue in another PR?

Comment (AllenFang) [reply src-react/containers/path.js]: Fix eslint on another PR, 
and I recommend use `resolve.modulesDirectories` to solve all the problem about require() relative path hell.

**Review by AllenFang — APPROVED** [review]

**Pull request merged**